insights/core/dr.py

```diff
@@ -414,7 +414,14 @@ def is_registry_point(component):
     return type(component).__name__ == "RegistryPoint"
 
 
-def get_registry_points(component):
+def is_datasource(component):
```

Comment:
@xiangce maybe it still make sense to have this function in `dr.py` but I've noticed that `insights/core/plugins.py` has [is_datasource](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/plugins.py#L370) function?

Reply:
@vdymna - Using the `plugins.is_datasource` would cause a `circular import issue`, since the `insights.core.plugins` imported the `insights.core.dr` ([here](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/plugins.py#L37)). And this is also the reason why it's implemented via checking the `comp_type.__name__ == 'datasource'` instead of `type(component) == datasource`.

Reply:
Gotcha, yeah I thought maybe that was the case.